corehq/motech/dhis2/schema.py

```diff
@@ -217,3 +210,11 @@ def get_program_owner_schema():
         "program": id_schema,
         "trackedEntityInstance": id_schema,
     }
+
+
+def get_tracked_entity_instance_schema():
+    return {
+        "trackedEntityInstance": id_schema,
+        SchemaOptional("programOwners"): [get_program_owner_schema],
```

Comment:
```suggestion
        SchemaOptional("programOwners"): [get_program_owner_schema()],
```
I wonder why this didn't cause `corehq/motech/dhis2/tests/test_schema.py` to fail. (Test data is in `corehq/motech/dhis2/tests/data/tracked_entity_instance_1.json`.) Could you add "potentialDuplicate" in that tracked entity JSON doc in a way that the test fails before this change, and passes after it?

Reply:
Yeah, interesting, I have added a test here and it did fail https://github.com/dimagi/commcare-hq/actions/runs/3226565110/jobs/5280236997. I have merged that branch here and it looks like it's now succeeding.